internal/server/server.go

```diff
@@ -195,6 +195,8 @@ func (ws *HTTPServer) Start() {
 	// Restore job status endpoint
 	mux.HandleFunc(ws.api("/restore/status/{jobId}"), ws.restoreStatusHandler)
 
+	mux.HandleFunc(ws.api("/restore/configuration/{name}/{timestamp}"), ws.restoreConfig)
```

Comment:
Add a comment explaining the endpoint. Also, having it under restore is misleading, since it returns data to the user.

Reply:
Yeah, but is it worth creating new category just for a single endpoint? We have everything about accessing backups under /restore